python/lsst/sims/maf/utils/getDateVersion.py

```diff
@@ -1,5 +1,6 @@
 import os
 import time
+import lsst.sims.maf
```

Comment:
If we move this import into the method ('getDateVersion') I *think* it only gets executed when you actually call getDateVersion. Otherwise, it's called on import of utils (as the __init__ there executes the imports of all the files it knows about). 

This seems like it might be a good thing, as we don't otherwise import lsst.sims.maf anywhere, and we only call getDateVersion once in normal operations.